### PR TITLE
Space page update for swecha

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
As per issue https://github.com/coderplex/coderplex/issues/162

### CODE REVIEW

**Comment by M-ZubairAhmed** [comment]
@duttakapil 
Please get the key for [Google Maps Embed API ](https://developers.google.com/maps/documentation/embed/guide), i will replace current with that

**Comment by duttakapil** [comment]
I'm seeing this when I open the deploy link
image

Browser : Firefox (latest)
OS : Windows (Latest)

**Comment by M-ZubairAhmed** [comment]
Ok can you get the key. It's the key fault

File: pages/space.js [hunk 1]
```diff
@@ -132,10 +140,19 @@ export default () => (
...
+        <Title inverted color="#222">
+          Location
+        </Title>
         <SubTitle inverted>
-          INR 1000/- per month. For membership mail us at <a href="mailto:space@codeplex.org">space@coderplex.org</a>
+          <a href="http://swecha.business.site/">Swecha</a>, Gachibowli<br />Hyderbad, India
         </SubTitle>
       </Container>
-    </PricingSection>
+      <iframe width="1000" height="450" frameBorder="0" src={spaceMapUrl} allowFullScreen />
```
Comment (vinaypuppal) [inline pages/space.js]: @M-ZubairAhmed Make map responsive for mobile devices

**Pull request merged**